lib/dialects/postgres/query-generator.js

```diff
@@ -524,6 +524,10 @@ const QueryGenerator = {
   },
 
   createFunction(functionName, params, returnType, language, body, options) {
+    if(!returnType) throw new Error("requires returnType");
```

Comment:
Spaces around if condition something like ``if (!returnType) {``, I feel you can combine all three `returnType` , `language` and `body` checks and throw a generic error message like "createFunction missing some parameters, did you forget to pass returnType, language or body."